jobs/software-developer.html

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+title: Software Developer
+permalink: /jobs/software-developer
+---
+
+<section>
+    <div class="container typography">
+        <h2>Software Developer</h2>
+        <p>We are looking for a full-stack software developer with interest and experience in user interface design.  You would be the seventh developer on our team, and the second with design skills.  The role is not exclusively front-end or design oriented, but we have grown to the point where we need more than one person who can handle design work when needed.</p>
+        <p>Some of our current frameworks are Ruby on Rails, React + Redux, and Xamarin.  It’s okay if you’re not an expert in those&mdash;we can train you.  We are especially interested in hiring developers who bring expertise in a different framework we may use in the future, and who are interested in learning new technologies, idioms, and patterns.</p>
+        <p>Finally, because (a) much of our work is client-facing and (b) we want to work efficiently as a remote team without resorting to heavy handed management, we look for candidates who are self-starters and have good communication and collaboration skills.</p>
+
+        <h3 id="process">Interview Process</h3>
+        <p>Our standard process is two phone interviews and a final onsite interview in Seattle, WA.  You may <a href="#customize">customize the process</a> if you don’t do well in phone interviews or are unable to travel.</p>
```

Comment:
Someone please test that this scroll ID still works. My local env is broken and I'm not wasting time with that right now.